spec-tests/src/main/java/org/candlepin/spec/bootstrap/client/request/Request.java

```diff
@@ -157,7 +157,7 @@ public Request setBasePath(String path) {
      * endpoint, minus the common components of the path that would be captured by the base path.
      * <p></p>
      * The endpoint path may optionally include parameters which will be replaced by the values set
-     * in any path parameters later. For example, in the path "owners/{owner_key}/products", the
+     * in any path parameters later. For example, in the path "/owners/{owner_key}/products", the
```

Comment:
It looks like we need the forward slash in the path, so I figured it would be a good idea to add that to the example in the javadoc

Reply:
:+1: